extension/src/main/java/io/quarkus/bot/build/reporting/extension/ReportingMavenExtension.java

```diff
@@ -44,11 +44,11 @@ public void afterSessionEnd(MavenSession session)
 
         BuildReport buildReport = new BuildReport();
 
-        Path topLevelProjectPath = result.getProject().getBasedir().toPath();
+        Path rootPath = Path.of("").toAbsolutePath();
```

Comment:
Coming here via the Quarkus main repo PR: Have you considered `MavenSession.getExecutionRootDirectory`?

Reply:
That could be another option. But isn't it the same thing in the end?

Reply:
And tbh, I'm not exactly sure if it would work as we would like for incremental builds given the root project is the first project of the list.

Reply:
I guess it's the same, but I don't have all the details on how Maven calculates it.
I was merely wondering why you did it yourself. 🙂

Reply:
Root means "currentProject" or something else?
I'm asking because there is a bug in Maven w.r.t currentProject that I had to work around in GIB.